sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -195,16 +195,21 @@ case class AlterTableRenameCommand(
       DDLUtils.verifyAlterTableType(catalog, table, isView)
       // If an exception is thrown here we can just assume the table is uncached;
       // this can happen with Hive tables when the underlying catalog is in-memory.
-      val wasCached = Try(sparkSession.catalog.isCached(oldName.unquotedString)).getOrElse(false)
-      if (wasCached) {
+      // If `optStorageLevel` is defined, the old table was cached.
+      val optStorageLevel = Try {
```

Comment:
Will `lookupCachedData` throw exception actually? If it's not cached, `None` should be returned. Isn't?

Reply:
`lookupCachedData` doesn't throw an exception. I believe the existing code is wrapped in `Try` because `isCached` is calling `spark.table` (could be related to "this can happen with Hive tables when the underlying catalog is in-memory"). So, I was keeping the same behavior.

Do you think `Try` can be removed?

Reply:
Yea I think we can remove it

Reply:
+1 to remove it. We use `lookupCachedData` in many places and I think we don't need to add `Try` here.